### ISSUE TITLE
FeedReader displays article recieved date/time rather than article published date/time ( owncloud )

### ISSUE THREAD

**Issue by clarjon1** [jangernert/FeedReader#169]
There is a discrepancy in how FeedReader displays the dates of an article. It appears that it stores/shows the date and time as when the application pulled the article, rather than using the information that owncloud stores with regards to the date and time.

Other applications, such as the owncloud news app for android, preserve this data.

Below is a screenshot of the discrepancy, including system date. As shown, the date/time that OwnCloud recieved the article is before the sync time of feedreader, which polls owncloud every 15 minutes, and uses that time.

![image](https://cloud.githubusercontent.com/assets/627845/17316860/9822e480-5845-11e6-96e7-efbcad3476e8.png)

**Comment by jangernert** [comment]
OwnCloud provides two timestamps, "pubDate" and "lastModified". Currently I use the "lastModified" stamp.

**Comment by jangernert** [comment]
So why do you think published date is better than modified? Please convince me =)

**Comment by clarjon1** [comment]
Say you've not had feedreader running for a couple of days. One of your RSS
feeds has time sensitive coupons and/or deals. Or has news stories, that do
not explicitly have the date in the article, using terms like "earlier
today", expecting the reader to reference the pubdate. In both of these
cases, one would look at the date of the article, and be misinformed about
its relevancy.

This also is how every RSS aggregation tool I've ever used has behaved. (
Owncloud news add-on, akregator, google reader back when it still existed,
It also apparently is leaving a bad taste in some users mouths: a friend of
mine told me after I recommended FeedReader that "it is unreliable - it
fucks up article dates."

TL;DR Don't sync for a week - oh wow, it looks like a ton of news happened
this morning, all these breaking news articles were written today!

**Comment by jangernert** [comment]
Like I said, it doesn't insert the date when the articles were synced, but the time they were last modified on owncloud. But I can change it if people think it's a big deal.
